.mailmap

```diff
@@ -1290,6 +1290,7 @@ sfoo <sfoohei@gmail.com>
 sharma-kunal <kunalsharma6914@gmail.com>
 shiksha11 <shiksharawat01@gmail.com> shiksha11 <33157995+shiksha11@users.noreply.github.com>
 shruti <shrutishrm512@gmail.com>
+shubhayu09 <guptashubhayu601@gmail.com> shubhayu09 <87441886+shubhayu09@users.noreply.github.com>
```

Comment:
@shubhayu09 what name do you want to be recorded in the AUTHORS file?

Reply:
> @shubhayu09 what name do you want to be recorded in the AUTHORS file?

Hey @oscarbenjamin ,
These are the correct details.